addons/project/models/project_task.py

```diff
@@ -1137,6 +1137,10 @@ def _change_operator(domain):
                     if isinstance(value, str) or (isinstance(value, list) and not all(isinstance(val, str) for val in value)):
                         new_domain.append(("name", op, value))
                     if isinstance(value, int):
+                        if op == "=":
+                            op = "in"
+                        if op == "!=":
+                            op = "not in"
```

Comment:
The warning is raised here: https://github.com/odoo/odoo/blob/4c1cab9e41931a9f88d4c31ca771003a3e5597ea/odoo/osv/expression.py#L688